lib/workers/repository/dependency-dashboard.ts

```diff
@@ -4,10 +4,12 @@ import { nameFromLevel } from 'bunyan';
 import { GlobalConfig } from '../../config/global';
 import type { RenovateConfig } from '../../config/types';
 import { getProblems, logger } from '../../logger';
+import type { PackageFile } from '../../modules/manager/types';
 import { platform } from '../../modules/platform';
 import { regEx } from '../../util/regex';
 import * as template from '../../util/template';
 import { BranchConfig, BranchResult } from '../types';
+import { getDepWarnings } from './onboarding/pr/errors-warnings';
```

Comment:
This does not seem to make sense. Why would/should the dashboard be sourcing dependency lookup warnings from the onboarding module?

Reply:
this is an example:
https://github.com/MaronHatoum/dashboard-dependency-lookup-warnings-reproduction/issues/7
maybe I should move `error-warnings` from `repository/onboarding/pr/` to `repository/`

Reply:
I will create a new refactor PR (to move `errors-warnings`)

Reply:
Yes, if you think that it's a good idea to reuse code with onboarding, first refactor that into a general location and then this PR can happen after that.